### PR TITLE
Test Temp Directory Spark Data Fixture

### PR DESCRIPTION
**Pull request by dpb-bah**
**Description:**
Simply adding a temporary directory to be used for Spark data storage for tests.

**Technical details:**
N/A

**Requirements for PR merge:**

1.  Unit & integration tests updated
2.  API documentation updated
3. [ ] Necessary PR reviewers:
    - [ ] Backend
    - [ ] Frontend <OPTIONAL>
    - [ ] Operations <OPTIONAL>
    - [ ] Domain Expert <OPTIONAL>
4.  Matview impact assessment completed
5.  Frontend impact assessment completed
6.  Data validation completed
7.  Appropriate Operations ticket(s) created
8.  Jira Ticket [DEV-123](https://federal-spending-transparency.atlassian.net/browse/DEV-123):
    -  Link to this Pull-Request
    -  Performance evaluation of affected (API | Script | Download)
    -  Before / After data comparison

**Area for explaining above N/A when needed:**
```
```

### CODE REVIEW

File: usaspending_api/etl/tests/integration/test_spark_app.py [hunk 1]
```diff
@@ -216,19 +233,27 @@ def test_spark_write_csv_app_run(spark: SparkSession, s3_unittest_data_bucket):
 
     df = spark.createDataFrame([Row(**data_row) for data_row in data])
     # NOTE! NOTE! NOTE! MinIO locally does not support a TRAILING SLASH after object (folder) name
-    df.write.option("header", True).csv(f"s3a://{s3_unittest_data_bucket}" f"/{CONFIG.DELTA_LAKE_S3_PATH}/write_to_s3")
+    csv_path = f"s3a://{s3_unittest_data_bucket}" f"/{CONFIG.DELTA_LAKE_S3_PATH}/write_to_s3"
+    if CONFIG.USE_FILESYSTEM_FOR_SPARK_DATA:
```
Comment (sethstoudenmier) [inline usaspending_api/etl/tests/integration/test_spark_app.py:237]: I would think that this would always be true living inside of tests.

Comment (collinwr) [reply usaspending_api/etl/tests/integration/test_spark_app.py:237]: @sethstoudenmier this should not be true when running in Travis. At that point, I think S3 can be used

Comment (dpb-bah) [reply usaspending_api/etl/tests/integration/test_spark_app.py:237]: Forgot to update `travis.yml` to set `USE_FILESYSTEM_FOR_SPARK_DATA` to `False`

**Comment by dpb-bah** [comment]
Closing. This was a band-aid fix for a bigger problem solved by https://github.com/fedspendingtransparency/usaspending-api/pull/3464

**Pull request closed without merging**